blocks/suggest/suggest.bemhtml

```diff
@@ -21,6 +21,7 @@ block('suggest')(
                     target : 'anchor',
                     autoclosable : true
                 },
+                attrs: { 'aria-hidden': "true" },
```

Comment:
Could you add this comment to the code, so it would be clear that is's a hack, not a over-engineering?